src/aioquic/asyncio/client.py

```diff
@@ -21,6 +21,7 @@
     create_protocol: Optional[Callable] = QuicConnectionProtocol,
     session_ticket_handler: Optional[SessionTicketHandler] = None,
     stream_handler: Optional[QuicStreamHandler] = None,
+    local_port: Optional[int] = None,
```

Comment:
The variable name is fine, could we have a docstring for it please?

Reply:
I don't think making it `Optional` is right, couldn't we initialize it straight to `0`?

Reply:
done.